ecosystem/sep-0001.md

```diff
@@ -80,7 +80,8 @@ ORG_PHONE_NUMBER_ATTESTATION | `https://` url | URL on the same domain as your `
 ORG_KEYBASE | string | A [Keybase](https://keybase.io/) account name for your organization. Should contain proof of ownership of any public online accounts you list here, including your organization's domain.
 ORG_TWITTER | string | Your organization's Twitter account
 ORG_GITHUB | string | Your organization's Github account
-ORG_OFFICIAL_EMAIL | string | An email where clients can contact your organization. Must be hosted at your `ORG_URL` domain.
+ORG_OFFICIAL_EMAIL | string | An email that business partners such as wallets, exchanges, or anchors can use to contact your organization. Must be hosted at your `ORG_URL` domain.
+ORG_SUPPORT_EMAIL | string | An email that users can use to request support regarding your Stellar assets or applications.
```

Comment:
❓ Should this also be hosted at the `ORG_URL`? I'm not advocating for it, more wondering why we require it on one email and not the other.

Reply:
I left that statement because I also wasn't sure why its there in the first place. It seems to be overly opinionated.

Reply:
It was added by Tom Q in https://github.com/stellar/stellar-protocol/commit/f7f69bb8f02142b38a0abb7bfc9483d36822feb0, which has no commit body or PR explaining. I think it was likely to encourage organizations to have official emails.

Reply:
Agreed, and not particularly useful.

Reply:
I could easily see a support email going to a ticketing system that doesn't use the same domain as `ORG_URL`.

Reply:
The goal might be authenticity and encouraging good best practices for building trust.

Reply:
Support email seems less critical to have this. I think it is fine to leave it off for it.